test/integration/export-lib.test.js

```diff
@@ -21,6 +21,7 @@ afterAll(() => {
 })
 
 test('It should export space when used as a library', () => {
+  console.log('!!!!!!!!', spaceId)
```

Comment:
would remove this :-)

Reply:
Oh my, thank you. I was in a hurry when I pushed my changes.